### PR TITLE
Feature/additional tests

### PR DESCRIPTION
**Pull request by FlynnOwen**
[x] Wrote test for feature

**Changes proposed:**
I've added additional unit tests to increase coverage.
FunctionFormatter and is_non_string_iterable had no tests, and I added additional tests for tautologies and contradictions to for the predicates module.

### CODE REVIEW

File: tests/test_formats.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import doctest
+import pytest
```
Comment (adnanhemani) [inline tests/test_formats.py:2]: Please don't add new import statements unless they're absolutely necessary

File: tests/test_util.py [hunk 2]
```diff
@@ -96,3 +96,14 @@ def test_proportions_from_distribution():
...
+
+def test_is_non_string_iterable():
+    is_string = 'hello'
+    assert ds.is_non_string_iterable(is_string) == False
+
+    is_list = [1, 2, 3]
+    assert ds.is_non_string_iterable(is_list) == True
+
+    is_int = 1
+    assert ds.is_non_string_iterable(is_int) == False
```
Comment (adnanhemani) [inline tests/test_util.py:109]: Can we add the remaining case that's not covered by tests right now: https://coveralls.io/builds/56773172/source?filename=datascience%2Futil.py#L257

**Review by adnanhemani — CHANGES_REQUESTED** [review]
Hi, thank you for this. Can you please address these comments and we should be okay to merge!

File: tests/test_util.py (hunk 2 above)
Comment (FlynnOwen) [reply tests/test_util.py:110]: Thanks for the feedback @adnanhemani, I've removed the import and updated with a unit test for the Sequence type.

**Review by adnanhemani — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**